anvil-rstudio-bioconductor/scripts/set_up_package_dir.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -e
+
+# This script needs to be separated from the Dockerfile because we mount persistent disks in the
+# /home/rstudio directory and when we mount the PD, everything in that directory gets wiped.
+# Therefore, we will run this script from Leonardo after the PD is mounted
+
+RSTUDIO_USER_HOME=/home/rstudio
+R_PATH=`R CMD printenv R_LIBS_USER | sed 's/~\///g'`
+BIOCONDUCTOR_VERSION=`printenv BIOCONDUCTOR_DOCKER_VERSION | sed 's/\(^[0-9].[0-9][0-9]\).*/\1/g'`
+R_PACKAGE_DIR=${RSTUDIO_USER_HOME}/${R_PATH}-${BIOCONDUCTOR_VERSION}
+
+mkdir -p ${R_PACKAGE_DIR}
+chmod a+rwx ${R_PACKAGE_DIR}
+
+echo R_LIBS=${R_PACKAGE_DIR} >> /usr/local/lib/R/etc/Renviron.site
```

Comment:
Related to above: does the `rstudio-user` user have permission to do this?

Reply:
It probably doesn't, but it's not `rstudio-user` here is it? @gpcarr Can confirm ?